pkg/config/default_common.go

```diff
@@ -5,3 +5,12 @@ package config
 
 // DefaultInitPath is the default path to the container-init binary.
 var DefaultInitPath = "/usr/libexec/podman/catatonit"
+
+var defaultUnixComposeProviders = []string{
+	"docker-compose",
+	"$HOME/.docker/cli-plugins/docker-compose",
```

Comment:
Who is going to expand the env? Your current code does not seem to do it.

Reply:
Nice catch!  Callers should use `os.ExpandEnv` on it (but the Podman PR does not yet).